pages/addBoard/index.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { ChangeEvent, useCallback, useEffect, useState } from "react";
+import styles from "./addBoard.module.css";
+import plusIcon from "@/public/images/icons/ic_plus.svg";
+import Image from "next/image";
+import { AddArticle } from "@/types/types";
+import { useRouter } from "next/router";
+
+const AddBoard = () => {
+  const router = useRouter();
+  const [values, setValues] = useState<AddArticle>({
+    title: "",
+    content: "",
+    image: null,
+  });
+  const [isDisabledSubmit, setIsDisabledSubmit] = useState(true);
+
+  const isDisabledCheck = useCallback(() => {
+    const { title, content } = values;
+    if (title && content) {
+      setIsDisabledSubmit(false);
+    } else {
+      setIsDisabledSubmit(true);
+    }
+  }, [values]);
```

Comment:
이미 관리하고있는 상태값으로 계산할 수 있는 영역이어서, disabled를 다룰때에 쓰일 state를 하나 더 만들지 않아도 될것같아요.

state 구조를 생각할 때에 state 생성 원칙이 [공식문서](https://ko.react.dev/learn/choosing-the-state-structure)에 있어요 여길 한번 참고 해 보세요.


```suggestion
const {title,content} = values;
const canSubmit = title&&content
// ...
// ...(중략)
<button className={styles.addBoardBtn} disabled={!canSubmit}>
```